cmd/bucket-replication.go

```diff
@@ -1600,7 +1600,11 @@ func proxyGetToReplicationTarget(ctx context.Context, bucket, object string, rs
 	return reader, proxyResult{Proxy: true}, nil
 }
 
-func getproxyTargets(ctx context.Context, bucket, object string, opts ObjectOptions) (tgts *madmin.BucketTargets) {
+func getProxyTargets(ctx context.Context, bucket, object string, opts ObjectOptions) (tgts *madmin.BucketTargets) {
+	if opts.VersionSuspended {
```

Comment:
opts.VersionSuspended  will not be set on GET/HEAD - may want to instead just do globalBucketVersioningSys.PrefixSuspended(bucket, object)

Reply:
done fixed @poornas